tensorflow/tools/pip_package/setup.py

```diff
@@ -109,8 +109,8 @@ def standard_or_nightly(standard, nightly):
     # dependencies on the release branch is updated to the stable releases (RC
     # or final). For example, 'keras-nightly ~= 2.14.0.dev' will be replaced by
     # 'keras >= 2.14.0rc0, < 2.15' on the release branch after the branch cut.
-    'tb-nightly ~= 2.18.0.a',
-    'keras-nightly >= 3.2.0.dev',
+    'tensorboard >= 2.18, < 2.19',
```

Comment:
Can you please also update the protobuf restriction above from `<5.0.0dev` to `<6.0.0dev`? We actually also need to do it in master earlier but missed that.

Reply:
okay. I will do it in another PR. Thanks.